renku/command/init.py

```diff
@@ -243,23 +227,42 @@ def _init(
 
     if branch_name:
         communication.echo(
-            "Project initialized.\n"
+            "Project initialized.\n"  # type: ignore
```

Comment:
this is weird. Why does this raise a type error?

Reply:
It's because of `project_properties.repository.head.reference` which can be `None` if we are in a detached HEAD state. We are not in that state, since we just created a master branch.
I've replaced `project_properties.repository.head.reference.name` with `HEAD` which has the same effect.